setup.py

```diff
@@ -16,7 +16,7 @@ def readme():
 
 vcs = p4 + github
 
-docs = ('sphinx', 'sphinx-argparse', 'sphinx_rtd_theme')  # This extra is required for RTD to generate documentation
+docs = ('sphinx=3.4.3', 'sphinx-argparse', 'sphinx_rtd_theme')  # This extra is required for RTD to generate documentation
```

Comment:
As far as I remember, it should be "==3.4.3"

Reply:
You are right: https://setuptools.readthedocs.io/en/latest/userguide/dependency_management.html#id4